build-system/server/lazy-build.js

```diff
@@ -15,18 +15,24 @@
  */
 'use strict';
 
+const {generateBundles} = require('../tasks/3p-vendor-helpers');
+
 const argv = require('minimist')(process.argv.slice(2));
 const {
   doBuildExtension,
   maybeInitializeExtensions,
   getExtensionsToBuild,
 } = require('../tasks/extension-helpers');
+const {buildVendorLazily} = require('../tasks/3p-vendor-helpers');
```

Comment:
Consistency nit: The "lazy" part comes from how the function is called and not what it does. Can we name this `doBuild3pVendor` to be consistent with `doBuildJs`?